packages-exp/auth-exp/src/platform_browser/persistence/indexed_db.ts

```diff
@@ -127,20 +127,11 @@ export async function _putObject(
   key: string,
   value: PersistenceValue | string
 ): Promise<void> {
-  const getRequest = getObjectStore(db, false).get(key);
-  const data = await new DBPromise<DBObject | null>(getRequest).toPromise();
-  if (data) {
```

Comment:
i think you still need this branch, my understanding is it's required for updating vs creating objects

Reply:
discussed IRL

Reply:
`The put() method of the IDBObjectStore interface updates a given record in a database, or inserts a new record if the given item does not already exist.` -- [MDN](https://developer.mozilla.org/en-US/docs/Web/API/IDBObjectStore/put)